clients/fides-js/src/components/Overlay.tsx

```diff
@@ -237,9 +241,6 @@ const Overlay: FunctionComponent<Props> = ({
             onClose: () => {
               setBannerIsOpen(false);
             },
-            onSave: () => {
-              setBannerIsOpen(false);
-            },
```

Comment:
`onSave` was just superfluous and confusing so I removed it in favor of `onClose` everywhere.